lib/AST/SwiftNameTranslation.cpp

```diff
@@ -159,7 +159,7 @@ swift::cxx_translation::getNameForCxx(const ValueDecl *VD,
   ASTContext& ctx = VD->getASTContext();
 
   if (const auto *Expose = VD->getAttrs().getAttribute<ExposeAttr>()) {
-    if (!Expose->Name.empty())
+    if (Expose->getExposureKind() == ExposureKind::Cxx && !Expose->Name.empty())
```

Comment:
Should we iterate over all of the `ExposeAttr`s for this decl, in case there are more than one?